modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+//networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sysctl-on-bonded-sriov-network_{context}"]
+= Configure sysctl on a bonded SR-IOV network 
+
+As a cluster administrator, set the interface specific sysctls on a bonded interface created from two SR-IOV interfaces by adding the tuning configuration to the optional `metaPlugins` parameter of the bond network attachment definition.
+
+
+[NOTE]
+====
+Do not edit `NetworkAttachmentDefinition` custom resources that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+To change all interface-level network sysctls create an additional SR-IOV network attachment with the Container Network Interface (CNI) tuning plug-in by using the following procedure.
```

Comment:
To change **all** interface-level network sysctls create an additional **SR-IOV network attachment** with the Container Network Interface (CNI) tuning plug-in by using the following procedure.

- all? - we can specify individual sysctls we want to be changed
- SR-IOV network attachment -  no such thing. We have the SriovNetwork for that, and the sriov operator creates a network-attachment-defintion from this
-